app-text/sigil/sigil-0.9.12.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
You should use repoman to fix this copyright header automatically for you, but for now, update to 
`# Copyright 1999-2019 Gentoo Authors`